src/ccl/sidebar/sidebar.component.ts

```diff
@@ -0,0 +1,40 @@
+/*
+ * Climate Change Lab
+ * Sidebar component
+ */
+import { Component, ViewEncapsulation, EventEmitter, Input, Output } from '@angular/core';
+
+import { IndicatorsService } from '../services/indicators.service';
+import { ChartService } from '../services/chart.service';
+
+/*
+ * Sidebar Component
+ * Populates sidebar with indicators and triggers adding charts
+ */
+
+@Component({
+  selector: 'sidebar',
+  encapsulation: ViewEncapsulation.None,
+  providers: [IndicatorsService, ChartService],
```

Comment:
Watch out. Whenever you add a provider to a component via this configuration, it instantiates a new instance of that provider at this level. So, if you did the same thing elsewhere in the app (which it looks like you're doing), you'd get two instances of the same provider. This can be good and bad depending on what you want. Bad if the provider in question is intended to be a singleton. This can cause subtle bugs, and is better explained here:

https://angular.io/docs/ts/latest/guide/hierarchical-dependency-injection.html#!#component-injectors

Read through to the end of the page. Might need to read the whole section for context. If this is confusing, I'm happy to explain tomorrow.